packages/tooling/__tests__/oas.test.js

```diff
@@ -185,6 +185,40 @@ describe('#findOperation()', () => {
     });
   });
 
+  it('should return result if path is slash', () => {
+    const oas = new Oas({
+      openapi: '3.0.0',
+      servers: [
+        {
+          url: 'https://example.com',
+        },
+      ],
+      paths: {
+        '/': {
+          get: {
+            responses: {
+              '200': {
+                description: 'OK',
+              },
+            },
+          },
+        },
+      },
+    });
+
+    const uri = 'https://example.com';
+    const method = 'get';
+
+    const res = oas.findOperation(uri, method);
+    expect(res.url).toStrictEqual({
```

Comment:
Just for sanity sake, can you also test that `res.operation` is filled with the right object?